src/python/foglamp_daemon.py

```diff
@@ -58,6 +58,47 @@ def safe_makedirs(directory):
         if not os.path.exists(directory):
             raise e
 
+def find_process_start_time(process_name):
+    """
+    Find Process start time
+
+    :param process_name: name of the process
+    :return: string start_time or None if no such process exists
+    """
+    f_ps = subprocess.Popen("ps h -C " + process_name + " -o lstart", shell=True, stdout=subprocess.PIPE)
+    start_time = f_ps.stdout.read().decode("utf-8")[:-1] # remove last \n character
```

Comment:
OK, this is one way to do it. I was thinking more along the lines of getting the time at startup and saving it. Then get the time now and do a simple subtraction.